modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -155,6 +155,15 @@ protected void cleanUp() {
 		_type = "link";
 	}
 
+	@Override
+	protected String getHydratedModuleName() {
+		if ((getAdditionalProps() != null) || (getPropsTransformer() != null)) {
+			return "frontend-taglib-clay/Link";
+		}
+
+		return null;
+	}
+
```

Comment:
You need to implement `prepareProps` to pass those props that are not in the dynamicAttributes already (those are added by the superclass)

```
@Override
	protected Map<String, Object> prepareProps(Map<String, Object> props) {
		props.put("icon", _icon);

		if (Validator.isNotNull(_label)) {
			props.put(
				"label",
				LanguageUtil.get(
					TagResourceBundleUtil.getResourceBundle(pageContext),
					_label));
		}

		return super.prepareProps(props);
	}
```